_maps/configs/pubby.json

```diff
@@ -1,15 +1,26 @@
 {
 	"map_name": "Pubby-class Light Carrier",
-	"map_short_name": "Punny-class",
+	"prefix": "NTSV",
+	"namelists": ["GENERAL", "BRITISH_NAVY", "MYTHOLOGICAL", "WEAPONS"],
+	"map_short_name": "Pubby-class",
 	"map_path": "_maps/shuttles/shiptest/whiteship_pubby.dmm",
 	"map_id": "whiteship_pubby",
 	"job_slots": {
-		"Captain": 1,
-		"Quartermaster": 1,
-		"Medical Doctor": 1,
-		"Station Engineer": 1,
-		"Shaft Miner": 2,
-		"Assistant": 2
+		"Captain": {
+			"outfit": "/datum/outfit/job/captain/nt",
+			"officer": true,
+			"slots": 1
+		},
+		"Shaft Miner": 1,
+		"Engineer": {
+			"outfit": "/datum/outfit/job/engineer/nt",
+			"slots": 1
+		},
+		"Pilot": {
+			"outfit":"/datum/outfit/job/cargo_tech/pilot",
```

Comment:
You'll want to make a new outfit datum here, this one appears to have been made for the Tartarus or some other ship and isn't really NT themed. My recommendation would be to make it security/nt/pilot, or to use the normal security/nt outfit while keeping the Pilot job title.

Some of the locker contents are also mismatched with the new Nanotrasen theme. Use under/rank/security/officer/nt for the officer jumpsuits in the pilot locker (bluesec shirts are the indie ones) and ditch the dusters, they're a strongly indie flavor. For the firearm locker, replace the M1911s and their magazines with Commanders and maybe give the captain an egun.

You also still have independent mining hardsuits in the SSUs, you can replace those with normal mining suits.